crates/web5/Cargo.toml

```diff
@@ -8,23 +8,31 @@ license-file.workspace = true
 
 [dependencies]
 base64 = { workspace = true }
+byteorder = "1.5.0"
 chrono = { workspace = true }
-keys = { path = "../keys" }
-dids = { path = "../dids" }
+did-jwk = "0.1.1"
+did-web = "0.2.2"
+ed25519-dalek = { version = "2.1.1", features = ["rand_core"] }
+getrandom = { version = "0.2.12", features = ["js"] }
 hex = "0.4"
 jsonpath-rust = "0.5.1"
 jsonschema = "0.17.1"
-jwt = { path = "../jwt" }
-jws = { path = "../jws" }
+k256 = { version = "0.13.3", features = ["ecdsa", "jwk"] }
 rand = { workspace = true }
+regex = "1.10.4"
 serde = { workspace = true }
 serde_json = { workspace = true }
 serde_with = { workspace = true }
+sha2 = "0.10.8"
+simple-dns = "0.7.0"
+ssi-core = "0.1.0"
+ssi-dids = "0.1.1"
```

Comment:
we still use this?

Reply:
yep, we're using ssi-dids and ssi-core for our did:web and did:jwk.